spec/unit/typesafe/config/concatenation_spec.rb

```diff
@@ -45,4 +45,383 @@
     expect(e.message).to include("null")
     expect(e.message).to include('{"x":"y"}')
   end
+
+  it "no arrays in string concat" do
```

Comment:
Yeah, now that I've reviewed it, I like it the way it is, it makes it super easy to review these and make sure we have all the tests. :+1: on being non-idiomatic.